src/type/definition.js

```diff
@@ -366,7 +366,7 @@ export type GraphQLScalarTypeConfig<TInternal, TExternal> = {
 export class GraphQLObjectType {
   name: string;
   description: ?string;
-  isTypeOf: ?GraphQLIsTypeOfFn;
+  isTypeOf: ?GraphQLIsTypeOfFn<*, *>;
```

Comment:
Any reason this isn't `<TSource, TContext>`? I'm updating the DefinitelyTyped @types/graphql and couldn't find this documented anywhere in Flow. Would be a great help to point me in the right direction.